pybrake/test_routes.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+from .route_metric import RouteMetric
+from .routes import _Routes, RouteStats, RouteStat
+
+
+def test_routes_performance_stats():
+    routes = _Routes(**{"config": {
+        "performance_stats": False,
+        "error_host": "https://api.airbrake.io",
+        "apm_host": "https://api.airbrake.io",
+    }})
+
+    metric = RouteMetric(method="GET", route="/test")
+    metric.status_code = 200
+    metric.content_type = "application/json"
+    metric.end_time = time.time()
+
+    assert routes.notify(metric) is None
+
+
+def _test_routes_notify():
+    routes = _Routes(**{"config": {
+        "performance_stats": True,
+        "error_host": "https://api.airbrake.io",
+        "apm_host": "https://api.airbrake.io",
+    }})
+
+    metric = RouteMetric(method="GET", route="/test")
+    metric.status_code = 200
+    metric.content_type = "application/json"
+    metric.end_time = time.time()
+
+    routes.notify(metric)
```

Comment:
`assert not none`?

Reply:
Because `performance_stats` is true in this testcase, no value will be returned.